tests/sdam/framework.rs

```diff
@@ -1,8 +1,8 @@
-use mongodb::{Client, ThreadedClient};
+use mongodb::Connector;
 use mongodb::Error::OperationError;
-use mongodb::connstring::{self, ConnectionString};
+use mongodb::connstring::ConnectionString;
```

Comment:
Changes to this file are a mix of using the updated API, the moving of `connstring::parse` to `ConnectionString::parse`, and from rustfmt.